docs/mobile/events.md

```diff
@@ -30,6 +30,16 @@ NOT be used together, each field MUST be used with its corresponding
 ### Event details
 
 <!-- semconv device.app.lifecycle(full) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
```

Comment:
does this empty block makes sense?

Reply:
It's empty because there are no attributes in the event.  THis is what we used to do, and this lets you know the autogen generated... nothing. 

I agree it's wierd but for this PR I was going to leave it as is and maybe fix in a different way.